### PR TITLE
aapt to ag: add Russian translation

### PR DESCRIPTION
**Pull request by Ryzhehvost**
work in progress, will add more translations later.

### CODE REVIEW

**Comment by sbrl** [comment]
Thanks, @Ryzhehvost! Just let us know when you're ready for us to review.

**Comment by Ryzhehvost** [comment]
It seems that other contributors are not checking for WIP PRs (#4905), so I guess sending translations one-by-one makes more sense. Please review what was done so far, I may send more PRs later if I'll have time.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

I think to get other Russian-speaking contributors to review this PR, you'll need to @mention them specifically and ask them.

We generally prefer for translations if you're translating lots of pages for you to put them in a single PR, because it makes the maintenance workload easier to manage :slightly_smiling_face:

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
Hi @Ryzhehvost, thanks for your submissions. Some adb pages are already in the `master` branch, could you please resolve the conflicts?

And in the another hand:
If I check the pages in `master`:
Screenshot_20201109_073131
But I can't resolve the encoding of your pages:
Screenshot_20201109_073227

Could you change the encoding of your translations to `UTF-8`? Thanks!

**Comment by Ryzhehvost** [comment]
Oh damn, totally forgot about the encoding, sorry. Fixed it, and resolved conflicts, please review.

File: pages.ru/common/adb-install.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+> Android Debug Bridge Install: Установка пакетов на эмулятор Android или подключенное устройство Android.
 > Больше информации: <https://developer.android.com/studio/command-line/adb>.
 
-- Передать Андроид приложение на эмулятор/устройство:
+- Установить приложение Android на эмулятор/устройство:
 
 `adb install {{путь/до/файла.apk}}`
 
-- Переустановить существующее приложение, оставляя его данные:
+- Пере-установить существующее приложение, сохранив его данные:
```
Comment (navarroaxel) [inline pages.ru/common/adb-install.md:10]: gTranslate prefers `Переустановить`, and the translation for `оставляя его данные` sounds better.
```suggestion
- Переустановить существующее приложение, оставляя его данные:
```

File: pages.ru/common/adb-shell.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
 # adb shell
 
-> Android Debug Bridge Shell: Запуск удалённых команд оболочки на экземпляре эмулятора Андроид или на подключённом Андроид устройстве.
+> Android Debug Bridge Shell: Запуск удалённой командной оболочки на эмуляторе Android или подключенном устройстве Android.
 > Больше информации: <https://developer.android.com/studio/command-line/adb>.
 
-- Запуск удалённой интерактивной оболочки на эмуляторе/устройстве:
+- Запустить удалённую интерактивную оболочку на эмуляторе/устройстве:
```
Comment (navarroaxel) [inline pages.ru/common/adb-shell.md:6]:
```suggestion
- Запустить удалённую интерактивную оболочку на эмуляторе или устройстве:
```

File: pages.ru/common/adb-shell.md [hunk 3]
```diff
@@ -27,10 +27,10 @@
 
 `adb shell pm clear {{пакет}}`
 
-- Запуск активности на эмуляторе/устройстве:
+- Запустить activity на эмуляторе/устройстве:
```
Comment (navarroaxel) [inline pages.ru/common/adb-shell.md:30]:
```suggestion
- Запустить activity на эмуляторе или устройстве:
```

File: pages.ru/common/adb-install.md (hunk 1 above)
Comment (Ryzhehvost) [reply pages.ru/common/adb-install.md:10]: maybe "оставив" would sound better than "оставляя"?

Comment (navarroaxel) [reply pages.ru/common/adb-install.md:10]: I don't speak Russian, I review this using gTranslator

Comment (Ryzhehvost) [reply pages.ru/common/adb-install.md]: Well, to be honest - all of the above are valid. "оставляя" is slightly inconsistent, but still technically correct. So I guess I'll make it "оставив".

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**